diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 225fdcaab5eb501d97387962ab8943c57d5d9a2b..0000000000000000000000000000000000000000
diff --git a/lib/move/move.js b/lib/move/move.js
index fa3ea618a29b6de8b76f4318c0b99700f878b39a..822400ede3a8dfbf744f65d56accb28ba05af75f 100644
--- a/lib/move/move.js
+++ b/lib/move/move.js
@@ -23,7 +23,8 @@ function move (src, dest, opts, cb) {
       if (err) return cb(err)
       mkdirp(path.dirname(dest), err => {
         if (err) return cb(err)
-        return doRename(src, dest, overwrite, cb)
+        // Is this dangerous? idk.
+        return doRename(src, dest, true, cb)
       })
     })
   })
